universe/Fleet.cpp

```diff
@@ -150,7 +150,8 @@ void Fleet::Copy(std::shared_ptr<const UniverseObject> copied_object, int empire
 
                 const std::list<int>& copied_fleet_route = copied_fleet->m_travel_route;
 
-                if (m_travel_route.empty() && !copied_fleet->m_travel_route.empty())
+                m_travel_route.clear();
+                if (!copied_fleet->m_travel_route.empty())
                     m_travel_route.push_back(moving_to);
```

Comment:
given the `this->m_travel_route = travel_route;` below, does any of this do anything?